src/modules/m_chanfilter.cpp

```diff
@@ -76,40 +95,62 @@ class ModuleChanFilter : public Module
 		cf.DoRehash();
 	}
 
+	void OnUserPart(Membership* memb, std::string& partmessage, CUList& except_list) CXX11_OVERRIDE
+	{
+		if (!memb)
+			return;
+
+		User* user = memb->user;
+		Channel* chan = memb->chan;
+		ChanFilter::ListItem* match = Match(user, chan, partmessage);
+		if (!match)
+			return;
+
+		// Match() checks the user is local, we can assume from here
+		LocalUser* luser = IS_LOCAL(user);
+
+		std::string oldreason(partmessage);
+		partmessage = "Filtered";
```

Comment:
In order to match the filter module:

```suggestion
		partmessage = "Reason filtered";
```